### PR TITLE
DateRange component

### PR DESCRIPTION
**Pull request by ao-alexi**

### CODE REVIEW

**Comment by ao-tigran** [comment]
@ao-alexi please resolve conflicts and change target to master.

**Comment by ao-alexi** [comment]
> @ao-alexi please resolve conflicts and change target to master.

@ao-tigran done

File: src/components/DateRange/DateRange.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React, { useRef, useEffect, useState } from "react";
+
+import DateTimePicker from "../DateTimePicker";
+
+const DateRange = props => {
+  const {
```
Comment (ao-tigran) [inline src/components/DateRange/DateRange.js]: Can we specify date vs time vs datetime?

Comment (ao-tigran) [reply src/components/DateRange/DateRange.js]: Can we have a min max?

File: src/components/DateRange/DateRange.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    startDate,
+    setStartDate,
+    endDate,
+    setEndDate,
+    startText,
```
Comment (ao-tigran) [inline src/components/DateRange/DateRange.js]: Are those labels?

**Comment by ao-tigran** [comment]
It also would be nice to have demo page.

File: src/components/DateRange/DateRange.js (hunk 1 above)
Comment (ao-alexi) [reply src/components/DateRange/DateRange.js]: yes, we can, i put some demo cases on HomeScreen (click login button for fake authorization)

File: src/components/DateRange/DateRange.js (hunk 2 above)
Comment (ao-alexi) [reply src/components/DateRange/DateRange.js]: no, just divs and text

**Pull request closed without merging**